### ISSUE TITLE
Does not work with electron-packager

### ISSUE THREAD

**Issue by TimoKunze** [megahertz/electron-log#154]
Hi,

I've problems to package my app with electron-packager. It throws an exception "Cannot throw module electron-log"
I create the package like this:
`electron-packager ./out/ my-app --asar --platform=win32 --arch=x64 --overwrite --ignore=\\.js\\.map --ignore=^/assets --ignore=^/custom-design --ignore=^/locales --ignore=./node_modules/electron-log --extra-resource=./node_modules/electron-log --out=dist`

Previously I did not pass the both parameters regarding electron-log, but the result has been the same. I use electron-log 3.0.7 and it is referenced in the dependencies section of package.json.
Any help appreciated.

**Comment by megahertz** [comment]
Have you tried to use other node modules with the same configuration? I don't think the module contains any special things which can lead to packaging problems.

**Comment by TimoKunze** [comment]
Hmm, I cannot remove electron-log from this project easily because it is used in quite a lot of places, but I'll try to find out.

**Comment by megahertz** [comment]
I mean, do you use another node modules in the project? If so, is the problem occurs only with electron-log?

**Comment by TimoKunze** [comment]
Well, in my package.json I reference these packages:
```
  "dependencies": {
    "electron-localshortcut": "^3.1.0",
    "electron-log": "^3.0.7",
    "ini": "^1.3.5",
    "lodash": "^4.17.15",
    "moment": "^2.24.0",
    "react": "^16.9.0",
    "react-dom": "^16.9.0",
    "react-intl": "^2.9.0",
    "react-scripts": "^3.1.1",
    "react-swipeable": "^5.4.0",
    "rimraf": "^2.7.1",
    "rxjs": "^6.5.2",
    "string-similarity": "^3.0.0",
    "xml2js": "^0.4.19",
    "zone.js": "^0.9.1"
}
```
So yes, I also use other modules in this project. The compiled exe stops after the first exception, so I guess to find out whether other node modules have the same problem, I would have to remove electron-log from the project.

**Comment by megahertz** [comment]
Try to import lodash before electron-log in your entry file

**Comment by TimoKunze** [comment]
Since I did not make much use of lodash, I rewrote these few lines of code and removed lodash. The result has been the same, but I started to shuffle around the order of the imports. Now it does not complain about electron-log anymore, but about ini:
```
import { App, Display, ipcMain, IpcMainEvent, screen } from "electron";
import * as fs from "fs";
import * as os from "os";
import * as path from "path";
import * as ini from "ini";
import moment from "moment";
import * as log from "electron-log";
// several imports for app-specific components
```
Is there any strategy that I should follow regarding the order of the imports? Currently I'm just trying around and if the order of imports in components referenced by the entry file also matters, I'll have tried all combinations around christmas. ^^
